crypto/perlasm/arm-xlate.pl

```diff
@@ -30,7 +30,7 @@
     else			{ ""; }
 };
 my $hidden = sub {
-    if ($flavour =~ /ios/)	{ ".private_extern\t".join(',',@_); }
+    if ($flavour =~ /ios|visionos/)	{ ".private_extern\t".join(',',@_); }
```

Comment:
If we really can't use `perlasm_format: "ios64"` then we need to document near the top of this script why visionos needs to be supported separately, but tvos and other variants don't. We should also try to upstream these changes to BoringSSL, if they are really needed, i.e. if we can't just use `"ios64"`.